game/resource/addon_english.txt

```diff
@@ -414,5 +419,10 @@
         "Script_6_Closing_2" "We will miss you young one. Make sure to say goodbye before you leave. You are now ready."
         "Script_6_Closing_3" "Don't feel bad if you lose, you're going to lose a lot."
         "Script_6_Closing_4" "That’s true, but Dota is one of the most rewarding games to play if you give it a chance. The map is yours my child. Good luck, and have fun!"
+        "Script_6_anger" "What’s popping gamer?! My name is AngerMania and I make lore videos on Youtube. If you feel overwhelmed with all this new knowledge and you’re worried that you’re not gonna ever get good at this game, you could always take a break from the gameplay and learn about the lore! My main content is videos about specific heroes, so maybe your favourite one has already been covered, or maybe learning the lore of a new hero will inspire you to try them out in the game. Check out the link for my playlist of all my Dota 2 hero lore videos. Good luck learning DOTA, maybe we will meet in game one day. PEACEE"
```

Comment:
Change Youtube to YouTube.
Change "PEACEE" to "Peace!".
Change DOTA to Dota (we use Dota anywhere else basically)